Gemfile.lock

```diff
@@ -8,7 +8,7 @@ GEM
   specs:
     coderay (1.1.2)
     diff-lcs (1.3)
-    google-protobuf (3.8.0)
+    google-protobuf (3.21.12)
```

Comment:
There are lots of changes: https://github.com/protocolbuffers/protobuf/compare/v3.8.0...v21.0